task.sh

```diff
@@ -6,44 +6,43 @@
 #   dir2
 #   dir3
 #       dir4
-mkdir -p /task/task
-mkdir dir1 dir2 dir3
-mkdir ./dir3/dir4
+# как же долго до меня доходит!!!
+mkdir -p task/dir1 task/dir2 task/dir3/dir4
 # изменяем текущую директорию на task
-cd /task/task
+cd ../task
 # создаём пустой файл task/dir2/empty
-touch ./dir2/empty
+touch dir2/empty
 # создаём файл task/dir2/hello.sh с таким содержанием:
 # #!/bin/bash
 # echo "$1, привет!"
 echo '#!/bin/bash
 echo "$1, привет!"' > dir2/hello.sh
 # устанавливаем для task/dir2/hello.sh права rwxrw-r--
-chmod -v 764 hello.sh
+chmod -v 764 dir2/hello.sh
 # сохраняем список файлов task/dir2 в task/dir2/list.txt
-ls dir2 > ./dir2/list.txt
+ls dir2 > dir2/list.txt
 # копируем содержимое каталога task/dir2 в каталог task/dir3/dir4
-cp ./dir2/* ./dir3/dir4
+cp dir2/* dir3/dir4
 # записываем в task/dir1/summary.txt список файлов с расширением *.txt
 # находящихся в task, включая поддиректории
-find ./task -name "*.txt" > ./dir1/summary.txt
+find ../task -name "*.txt" > dir1/summary.txt
```

Comment:
Тут мы сначала вышли из папки не уровень выше, а потом в нее вернулись. Можно просто 
```suggestion
find . -name "*.txt" > dir1/summary.txt
```